.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: rust
+
+rust:
+  - stable
+  - beta
+  - nightly
+matrix:
+  allow_failures:
+    - rust: nightly
+
+os:
+  - linux
+  - osx
```

Comment:
I think it's overkill to do OS X builds when none of the code in this crate is platform-dependent.